content/docs/command-reference/metrics/index.md

```diff
@@ -63,33 +33,8 @@ stages:
           cache: false
 ```
 
-> `cache: false` above specifies that `summary.json` is not tracked or
-> <abbr>cached</abbr> by DVC (`-M` option of `dvc run`). These metrics files are
-> normally committed with Git instead. See `dvc.yaml` for more information on
-> the file format above.
```

Comment:
Looks like this note should stay? It refers to the code block above, which is left in the ref.

Reply:
Ah never mind I see we rewrote this as a regular p.